src/tree.rs

```diff
@@ -714,7 +717,8 @@ impl<T> Tree<T> {
     /// # }
     /// ```
     ///
-    pub fn sort_children_by_key<B, F>(&mut self, node_id: &NodeId, mut f: F) -> Result<(), NodeIdError>
+    pub fn sort_children_by_key<B, F>(&mut self, node_id: &NodeId, mut f: F)
+                                      -> Result<(), NodeIdError>
         where B: Ord, F: FnMut(&Node<T>) -> B
     {
```

Comment:
I am not sure, if I know rustfmt good enough for this specific. Of course it has to do with the maximum line length, but in case you want to keep that, I am not sure, if you can control the return value indentation.
You could probably fill an issue on rustfmt. If there is an option and we cannot find it, this is at least a documentation/discoverability problem.